sources/tests/workflow-docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+
+services:
+
+  zeebe-test:
```

Comment:
Is there a need for the host/service names to be appended with "-test" or only the `container_name`?

Reply:
It is only the container name. Just for the sake of keeping the names in sync, I used service name with -test.

![image](https://user-images.githubusercontent.com/54600590/225161952-23489361-56a4-46e6-b029-22108e9a246b.png)